src/core_plugins/kibana/public/dashboard/components/full_screen_placeholder.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import chrome from 'ui/chrome';
+
+import {
+  KuiKeyboardAccessible,
+} from 'ui_framework/components';
+import {
+  keyCodes,
+} from 'ui_framework/services';
+
+export class FullScreenModePlaceholder extends PureComponent {
+
+  onKeyDown = (e) => {
```

Comment:
Think it might be worth adding a Jest test file for this component, to do a basic snapshot and also test that the callback is called when the ESCAPE key is pressed? There are some Jest test guidelines at https://github.com/elastic/eui/blob/master/wiki/testing.md#anatomy-of-a-test in case you find them useful.